components/raftstore-v2/src/batch/store.rs

```diff
@@ -432,6 +439,11 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
             ),
         );
 
+        let engine_gc_scheduler = workers.engine_gc_worker.start_with_timer(
```

Comment:
Confusing name.

Reply:
I was planning to put all engine related works into it, like tablet gc, log gc, purge etc.